k8s/helm/templates/deployment.yaml

```diff
@@ -23,6 +23,7 @@ spec:
       labels:
         app.kubernetes.io/name: {{ .Chart.Name }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+        date: "{{ .Release.Time.Seconds }}"
```

Comment:
Was this so that helm upgrade a deployment with the same image tag?

Reply:
Yes